mindone/trainers/came.py

```diff
@@ -0,0 +1,316 @@
+                v_col.append(
+                    Parameter(
+                        ops.zeros(x.shape[:-2] + x.shape[-1:], dtype=ms.float32),
+                        name=x.name + "_v_col",
+                        requires_grad=False,
+                    )
+                )
+                v_res_row.append(
+                    Parameter(
+                        ops.zeros(x.shape[:-1], dtype=ms.float32),
+                        name=x.name + "_v_res_row",
+                        requires_grad=False,
+                    )
+                )
+                v_res_col.append(
+                    Parameter(
+                        ops.zeros(x.shape[:-2] + x.shape[-1:], dtype=ms.float32),
+                        name=x.name + "_v_res_col",
+                        requires_grad=False,
+                    )
+                )
+                v.append(
+                    Parameter(
+                        ops.zeros((1,), dtype=ms.float32),
+                        name=x.name + "_v",
+                        requires_grad=False,
+                    )
+                )
+            else:
+                v_row.append(
+                    Parameter(
+                        ops.zeros((1,), dtype=ms.float32),
+                        name=x.name + "_v_row",
+                        requires_grad=False,
+                    )
+                )
+                v_col.append(
+                    Parameter(
+                        ops.zeros((1,), dtype=ms.float32),
+                        name=x.name + "_v_col",
+                        requires_grad=False,
+                    )
+                )
+                v_res_row.append(
+                    Parameter(
+                        ops.zeros((1,), dtype=ms.float32),
+                        name=x.name + "_v_res_row",
+                        requires_grad=False,
+                    )
+                )
+                v_res_col.append(
+                    Parameter(
+                        ops.zeros((1,), dtype=ms.float32),
+                        name=x.name + "_v_res_col",
+                        requires_grad=False,
+                    )
+                )
+                v.append(
+                    Parameter(
+                        ops.zeros_like(x, dtype=ms.float32),
+                        name=x.name + "_v",
+                        requires_grad=False,
+                    )
+                )
+
+        self.v_row = ParameterTuple(v_row)
+        self.v_col = ParameterTuple(v_col)
+        self.v_res_row = ParameterTuple(v_res_row)
+        self.v_res_col = ParameterTuple(v_res_col)
+        self.v = ParameterTuple(v)
+
+        self.m = ParameterTuple(
+            [
+                Parameter(
+                    ops.zeros_like(x, dtype=ms.float32),
+                    name=x.name + "_m",
+                    requires_grad=False,
+                )
+                for x in self._parameters
+            ]
+        )
+
+    @ms.jit
```

Comment:
why add jit?

Reply:
almost all mindspore optimizers implementation using `jit` to accelerate training. Here it is following the same practice.